index.js

```diff
@@ -1,4 +1,5 @@
 /* @flow strict-local */
+import 'react-native-gesture-handler';
```

Comment:
See React Navigation's setup instructions at
      https://reactnavigation.org/docs/5.x/getting-started#installing-dependencies-into-a-bare-react-native-project :
    
      To finalize installation of `react-native-gesture-handler`, add
      the following at the **top** (make sure it's at the top and
      there's nothing else before it) of your entry file, such as
      `index.js` or `App.js`:
    
      ```js
      import 'react-native-gesture-handler';
      ```
    
      > Note: If you are building for Android or iOS, do not skip this
      > step, or your app may crash in production even if it works fine
      > in development. This is not applicable to other platforms.

Wacky.

Probably should include a code comment here mentioning that quirky constraint, like:
```suggestion
// React Navigation requires this react-native-gesture-handler import,
// as the very first import of this entry-point file.  See our #5373.
import 'react-native-gesture-handler';
```